vm/simple-jit/ebpf_vm.c

```diff
@@ -989,14 +994,16 @@ int ebpf_exec(const struct ebpf_vm *vm, void *mem, size_t mem_len,
 					__ATOMIC_RELAXED);
 				break;
 			}
-			// case EBPF_CMPXCHG: {

```

Comment:
why commented out these code?

Reply:
They were introduced in a previous commit, I'll check it